adhocteam/Head-Start-TTADP
# Example widget

## Description of change

Adds an example widget that roughly matches the data in the `TTA overview` widget. The focus of this PR is around the pattern this widget sets and not on the widget itself.

## How to test

If you have a large number of activity reports locally you can test there by going to `widgets`. I loaded legacy reports and used those to test by importing several regional CSVs into region 1. Note if you do that some fields will be zero (objectives complete and grantees served) because the legacy import doesn't import that data. Let me know if you need the legacy report CSV files and I can send them via keybase. ~I'm hoping a better bet is to try it out on sandbox but I can't currently login to sandbox to see how many reports have been created.~ Scratch that there are 0 reports in sandbox, easiest way is probably to just import the legacy reports locally.

~Note the coverage limits have been set to 0 so it will deploy to sandbox. I didn't want to start writing tests until I got some feedback. If you see me set this PR to `Ready for review` without changing those limits back be sure to yell at me!~

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-120

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
